src/barril/units/unit_system_interface.py

```diff
@@ -0,0 +1,124 @@
+from oop_ext.foundation import callback
+from oop_ext.interface import Attribute, Interface
+
+
+class IUnitSystem(Interface):
```

Comment:
Anyone know why do we have this interface? Why `UnitSystem` is not just a configurable object?

Reply:
@kfasolin you know the answer? I'm not sure.

Reply:
Nop, famous "it was already there when I arrived". This code is from 2008, maybe interfaces were the way to go by then.

Reply:
I believe that we have an interface because we have an UnitSystemManager with methods such as AddUnitSystem and SetCurrent. So different  projects can add its unit systems (kraken / rocky for example can add unit systems dinamically)